### PR TITLE
Vendor gardener@v1.6.5

### PR DESCRIPTION
**Pull request by plkokanov**
**What this PR does / why we need it**:
Vendors gardener 1.6.5 so that the os-ubuntu-alicloud extension can make use of the Restore and Migrate operation implementations from the operatingsystemconfig generic actuator.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Vendor gardener@v1.6.5
```

### CODE REVIEW

**Review by timebertt — DISMISSED** [review]
/lgtm

Same for the CI failure like here: https://github.com/gardener/gardener-extension-os-coreos/pull/12#pullrequestreview-441385931

File: go.mod [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 
 replace (
-	k8s.io/api => k8s.io/api v0.16.8 // 1.16.8
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.16.8 // 1.16.8
-	k8s.io/apimachinery => k8s.io/apimachinery v0.16.8 // 1.16.8
-	k8s.io/client-go => k8s.io/client-go v0.16.8 // 1.16.8
-	k8s.io/code-generator => k8s.io/code-generator v0.16.8 // 1.16.8
-	k8s.io/component-base => k8s.io/component-base v0.16.8 // 1.16.8
+	k8s.io/api => k8s.io/api v0.17.6 // 1.17.6
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.6 // 1.16.8
```
Comment (ialidzhikov) [inline go.mod]: Nit

```suggestion
	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.6
```

**Comment by vpnachev** [comment]
/needs rebase
to fix failing CI check

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**